docs/example.md

```diff
@@ -1,24 +1,25 @@
 # Full workflow tutorial
 
-This example demonstrates how to export application deployed on Openshift,
-to Nulecule specification and then import that application to Kubernetes.
+This example demonstrates how to export application deployed on Openshift
+to Nulecule specification and then import that application to Kubernetes and/or OpenShift.
 
 This is a 3 step tutorial:
     1. Deploy a sample application on Openshift.
     2. Export the application using openshift2nulecule.
     3. Import the exported artifacts into Kubernetes using Atomic App.
```

Comment:
Import the exported artifacts into Kubernetes and/or OpenShift using Atomic App.